bullet-featherstone/src/KinematicsFeatures.cc

```diff
@@ -61,14 +61,30 @@ FrameData3d KinematicsFeatures::FrameDataRelativeToWorld(
     // Groups are always Models underneath. If we ever support frame semantics
     // for more than links, models, and free groups, then this implementation
     // needs to change.
-    model = this->FrameInterface<ModelInfo>(_id);
+    // model = this->FrameInterface<ModelInfo>(_id);
+    // I commented this line because it's returning a bad object.
+    // I'm not able to access getBaseWorldTransform
+    for (auto const [key, value] : this->links)
+    {
+      for (unsigned int i = 0; i < value->collisionEntityIds.size(); ++i)
+      {
+        if (value->collisionEntityIds[i] == _id.ID())
```

Comment:
Are you seeing a case where the incoming `_id` is for a collision entity? The [kinematics feature list](https://github.com/gazebosim/gz-physics/blob/166ff209ff4e74c2ab8e5daed1214b2e4aa6e786/bullet-featherstone/src/KinematicsFeatures.hh#L31-L33) doesn't include [`ShapeFrameSemantics`](https://github.com/gazebosim/gz-physics/blob/166ff209ff4e74c2ab8e5daed1214b2e4aa6e786/include/gz/physics/FrameSemantics.hh#L189) so it's not supposed to be possible for a user to ask for the frame data of a collision object.

If you're seeing that happen then it probably indicates an error in the template system. If you can point me at a way to recreate it, that would be very helpful.